qcelemental/molutil/__init__.py

```diff
@@ -1 +1,3 @@
 from .align import B787, kabsch_align, compute_scramble
+from .test_align import test_hessian_align, test_vector_gradient_align
```

Comment:
this shouldn't be needed as we don't want the tests to be part of the module proper. if you let me know the error message that led to this addition, I can probably suggest an alternate solution.

Reply:
I just wanted to be able to run the tests from an external directory.  If they are not here, how do you do that?   I was just testing with:
> sys.path.insert(0, '/home/rking/programs/QCElemental')
> import qcelemental as qcel
>qcel.molutil.test_vector_gradient_align()

Reply:
Try just “pytest” once “import qcelemental as qcel” works.